test/e2e/suite/serving/cainjector.go

```diff
@@ -153,22 +154,14 @@ var _ = framework.CertManagerDescribe("CA Injector", func() {
 				injectable, cert := generalSetup(test.makeInjectable("changed"))
 
 				By("changing the name of the corresponding secret in the cert")
-				retry.RetryOnConflict(retry.DefaultRetry, func() error {
```

Comment:
These retries were originally added to deal with update conflicts https://github.com/cert-manager/cert-manager/pull/4925